packages/sources/kaiko/src/adapter.ts

```diff
@@ -16,33 +16,20 @@ const customParams = {
   includes: false,
 }
 
+const symbolUrl = (from: string, to: string) =>
+  to.toLowerCase() === 'eth'
+    ? directUrl(from, to)
+    : `/spot_exchange_rate/${from.toLowerCase()}/${to.toLowerCase()}`
+
+const directUrl = (from: string, to: string) =>
+  `/spot_direct_exchange_rate/${from.toLowerCase()}/${to.toLowerCase()}`
+
 export const execute: ExecuteWithConfig<Config> = async (request, config) => {
-  const validator = new Validator(request, customParams)
+  const validator = new Validator(request, customParams, { name: "kaiko" })
```

Comment:
What does the "name" option here do?
Regardless, if we need the adapter name we should use the name that's exported in `index.ts` or `config.ts`.

Reply:
Yeah, this code is unnecessary it seems, will delete.